src/commands.c

```diff
@@ -2632,9 +2632,7 @@ IMPLEMENT_COMMAND (exit)
         gtk_widget_destroy (GTK_WIDGET (uzbl.gui.plug));
     }
 
-    if (uzbl.state.gtk_started) {
-        gtk_main_quit ();
-    }
+    gtk_main_quit ();
```

Comment:
Does this crash if you put `exit` into the configuration file? GTK isn't started before we read the configuration file.